lib/runner.ex

```diff
@@ -30,14 +42,16 @@ defmodule Runner do
     modules
     |> Enum.drop_while( &(&1 != start_module))
     |> Enum.take_while( &(run_module(&1) == :passed))
+    |> done?
   end
 
+  defp done?(modules), do: modules == @modules
```

Comment:
Dang you're absolutely right... What if we had a process that kept a running list of complete modules referred to that for doneness?